src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerGenotypingEngine.java

```diff
@@ -683,4 +673,10 @@ public PloidyModel getPloidyModel() {
     public IndependentSampleGenotypesModel getGenotypingModel() {
         return genotypingModel;
     }
+
+    // check whether all alleles of a vc, including the ref but excluding the NON_REF allele, are one base in length
+    protected static GenotypeLikelihoodsCalculationModel getGLModel(final VariantContext vc) {
+        final boolean isSNP = vc.getAlleles().stream().filter(a -> !a.isSymbolic()).allMatch(a -> a.length() == 1);
```

Comment:
Yes.  `VariantContext::isSNP` only returns `true` when every alt allele is a SNP, and therefore returns false when there is a NON_REF symbolic allele.